src/views/clusteroverview/overview/components/getting-started-card/related-operators-section/RelatedOperatorsSection.tsx

```diff
@@ -56,7 +58,7 @@ const RelatedOperatorsSection: React.FC = () => {
       title={t('Related operators')}
       titleColor={'var(--co-global--palette--orange-400)'}
       description={t('Ease operational complexity with virtualization by using Operators.')}
-      links={links}
+      links={isAdmin ? links : []}
```

Comment:
```suggestion
      links={operatorDescriptions}
```
this are not the actual links, they include the description and the links :-)
the descritions should stay, only the links should be removed, something like:
``` ts
{
      id: 'openshift-data-foundation',
      title: t('OpenShift Data Foundation'),
      href: isAdmin ? '/operatorhub/all-namespaces?keyword=OCS' : null,
    },
```

Reply:
( and check that `GettingStartedSectionContents` can handle null links gracefully )

Reply:
yes, if no `href` we can remove the '->' and the link behaviour